source/index.ts

```diff
@@ -6,14 +6,17 @@ import type {ky as KyInterface} from './types/ky.js';
 import type {Input, Options} from './types/options.js';
 import {validateAndMerge} from './utils/merge.js';
 
+type Mutable<T> = {
+	-readonly[P in keyof T]: T[P]
+};
```

Comment:
It's needed for `KyInterface` as it has `readonly stop` which needs to make mutable below (to be able to assign the value)